src/transforms/concat.rs

```diff
@@ -0,0 +1,311 @@
+use super::{BuildError, Transform};
+use crate::{
+    event::Event,
+    topology::config::{DataType, TransformConfig, TransformDescription},
+};
+use regex::bytes::Regex;
+use serde::{Deserialize, Serialize};
+use string_cache::DefaultAtom as Atom;
+
+use lazy_static::lazy_static;
+
+#[derive(Deserialize, Serialize, Debug)]
+#[serde(deny_unknown_fields)]
+pub struct ConcatConfig {
+    pub target: Atom,
+    pub joiner: String,
```

Comment:
Might be worth making this an `Option` and defaulting to a space when it's not specified.

Reply:
Done :)